test/odbc/main.cpp

```diff
@@ -1,8 +1,26 @@
 #include <gtest/gtest.h>
+#include <map>
+#include <string>
 #include "odbc_handler.h"
 
+using std::map;
+using std::string;
+
 int main(int argc, char **argv) {
+  OdbcHandler odbcHandler;
+  map<ServerType, ConnectionStringObject> drivers = odbcHandler.getOdbcDrivers();
```

Comment:
Opinion: Would we be able to handle this logic in a new class? It feels a bit off to use an instance of Odbchandler to simply parse strings, but not use any ODBC calls. I think the ODBC handler class should be used to simplify ODBC calls and not a config file parser.

Reply:
Done. Please review.